templates/prescription/general_list.html

```diff
@@ -161,183 +161,181 @@
-                            <a target="_blank" id="multiprint" href="{$GLOBALS.webroot}/{$GLOBALS.baseModDir}{$GLOBALS.zendModDir}/public/prescription-pdf-template/{$GLOBALS.rx_zend_pdf_action}?id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary">{xlt t='Download'} ({xlt t='PDF'})</a>
+        {if $prescriptions}
+        <h3>{xlt t='List'}</h3>
+        <div id="prescription_list">
+            <form name="presc">
+                <div id="print_links">
+                    <table class="w-100">
+                        <tr>
+                            <td>
+                                <table>
+                                    <tr>
+                                        <td>
+                                        {if $GLOBALS.rx_zend_pdf_template}
+                                            <a target="_blank" id="multiprint" href="{$GLOBALS.webroot}/{$GLOBALS.baseModDir}{$GLOBALS.zendModDir}/public/prescription-pdf-template/{$GLOBALS.rx_zend_pdf_action}?id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary">{xlt t='Download'} ({xlt t='PDF'})</a>
+                                        {else}
+                                            <a id="multiprint" href="{$CONTROLLER}prescription&multiprint&id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary btn-download">{xlt t='Download'} ({xl t='PDF'})</a>
+                                        {/if}
+                                        </td>
+                                        <td>
+                                        {if $GLOBALS.rx_zend_html_template}
+                                            <a target="_blank" id="multiprintcss" href="{$GLOBALS.webroot}/{$GLOBALS.baseModDir}{$GLOBALS.zendModDir}/public/prescription-html-template/{$GLOBALS.rx_zend_html_action}?id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary">{xl t='View Printable Version'|text} ({xlt t='HTML'})</a>
+                                        {else}
+                                        <!-- TajEmo work by CB 2012/06/14 02:16:32 PM target="_script" opens better -->
+                                            <a target="_script" id="multiprintcss" href="{$CONTROLLER}prescription&multiprintcss&id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary btn-print">{xlt t='View Printable Version'} ({xlt t='HTML'})</a>
+                                        {/if}
+                                        </td>
+                                        {if $GLOBALS.rx_use_fax_template}
+                                        <td style="border-style:none;">
+                                            <a id="multiprintToFax" href="{$CONTROLLER}prescription&multiprintfax&id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary btn-download">{xlt t='Download'} ({xlt t='Fax'})</a>
+                                        </td>
```

Comment:
Hi @tywrenn 

I think this `style="border-style:none;"` maybe can remove or change to `border-0` ?

Reply:
@stu01509 Yeah change it to `border-0`